src/locale/en.yml

```diff
@@ -52,6 +52,7 @@ feat:
       type: Type to filter content
     lastDayWithEvents: There {numEvents, plural, one {was one event} other {were
       {numEvents} events}} on the last active day
+    loadMore: Load more
```

Comment:
This was unrelated to this PR, but it's correct to have it in the file (eventually) so I don't mind.